build.gradle

```diff
@@ -170,7 +170,7 @@ repositories {
 }
 
 def versions = [
-        springBootVersion: '2.6.3',
```

Comment:
hi Sunil, looks like https://github.com/hmcts/probate-business-service/blob/17aa1043cabe0c15dbac918e6254edfe76b2dccc/build.gradle#L11 change is missed for this repo.